cmd/chart.go

```diff
@@ -81,9 +81,9 @@ func newChartMoveCmd() *cobra.Command {
 	f.UintVar(&retries, "retries", defaultRetries, "number of times to retry push operations")
 	f.StringVar(&output, "out", "*.relocated.tgz", "name of the resulting chart")
 
-	f.StringVar(&toArchive, "to-archive", "", "save the chart and all its dependencies to an intermediate archive tarball")
+	f.StringVar(&toArchive, "to-intermediate-bundle", "", "save the chart and all its dependencies to an intermediate archive tarball")
```

Comment:
what about keeping the previous flag for compatibility but leave it hidden, basically mark is as deprecated.

Reply:
ok